pypackify.py

```
@@ -9,8 +11,23 @@ def load_config(config_path="config.json"):
     with open(config_path, "r") as file:
         return json.load(file)
 
+def load_license_text(license_name):
```

Ruff (0.8.2)

Missing return type annotation for public function `load_license_text`

(ANN201)